src/components/OverviewMap/OverviewMap.tsx

```diff
@@ -182,6 +205,7 @@ const OverviewMap = ({ isPublic = false, ...rest }) => {
 
   // request data on mount
   useEffect(() => {
+    // eslint-disable-next-line @typescript-eslint/no-floating-promises
```

Comment:
How about disabling this in eslintrc.js? Void promises should be allowed in our code.
```
'@typescript-eslint/no-floating-promises': [
          'error',
          {
            ignoreVoid: true,
          },
        ],
```